packages/shader-lab/src/common/BaseScanner.ts

```diff
@@ -65,24 +71,18 @@ export default class BaseScanner {
     }
   }
 
-  /**
-   * @internal
-   */
   _advance(): void {
     if (this.isEnd()) {
       return;
     }
 
-    this._currentIndex++;
-
-    // #if _EDITOR
     if (this.getCurChar() === "\n") {
```

Comment:
Why delete _VERBOSE

Reply:
Done, all column and line properties are move to verbose mode.